photutils/psf/photometry.py

```diff
@@ -0,0 +1,426 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Module which provides classes to perform PSF Photometry"""
+
+from __future__ import division
+import abc
+import numpy as np
+from astropy.extern import six
+from astropy.table import Table, vstack
+from astropy.modeling.fitting import LevMarLSQFitter
+from astropy.nddata.utils import overlap_slices, NoOverlapError
+from astropy.stats import gaussian_sigma_to_fwhm
+from ..psf import subtract_psf
+from ..aperture import CircularAperture, aperture_photometry
+
+
+__all__ = ['DAOPhotPSFPhotometry']
+
+
+class DAOPhotPSFPhotometry(object):
+    """
+    This class implements the DAOPHOT algorithm proposed by Stetson
+    (1987) to perform point spread function photometry in crowded fields,
+    which consists basically in applying the loop FIND, GROUP, NSTAR,
+    SUBTRACT, FIND until no more stars are detected or a given number of
+    iterations is reached.
+    """
+
+    def __init__(self, group, bkg, psf, fitshape, find=None,
+                 fitter=LevMarLSQFitter(), niters=3, aperture_radius=None):
+        """
+        Attributes
+        ----------
+        find : callable or instance of any StarFinderBase subclasses
```

Comment:
Attributes should be documented in the same order they appear in the function signature.